src/main/java/de/tum/in/www1/artemis/service/CourseExportService.java

```diff
@@ -145,7 +145,7 @@ private void exportCourseExams(Course course, String outputDir, List<String> exp
             Files.createDirectory(examsDir);
 
             // Lazy load the exams of the course.
-            var courseWithExams = courseRepository.findWithEagerLecturesAndExamsById(course.getId());
+            var courseWithExams = courseRepository.findWithEagerLecturesAndLectureUnitsAndExamsById(course.getId());
```

Comment:
Is it really necessary to load lectures and lecture units here?

Reply:
Changed it in: 9635d1aa97fc729f44053a54648ed2b8f6a95718